noxfile.py

```diff
@@ -20,7 +20,7 @@ def lint(session):
     session.run("pre-commit", "run", "--all-files", *session.posargs)
 
 
-@nox.session(python=ALL_PYTHONS)
+@nox.session(python=ALL_PYTHONS, reuse_venv=True)
```

Comment:
I think this is safe. Let's try it, anyway; it saves quite a few (15-20) seconds for me.